modules/experimental-layers/src/gpu-grid-layer/gpu-grid-cell-layer-vertex.glsl.js

```diff
@@ -97,16 +97,11 @@ void main(void) {
   vec4 position_worldspace;
   gl_Position = project_position_to_clipspace(extrudedPosition, extrudedPosition64xyLow, offset, position_worldspace);
 
-  float lightWeight = 1.0;
-
-  if (extruded > 0.5) {
-    lightWeight = lighting_getLightWeight(
-      position_worldspace.xyz, // the w component is always 1.0
-      normals
-    );
+   if (extruded > 0.5) {
```

Comment:
Note: Worth auditing if we want lighting to be controlled via extrusion prop or separately.

Reply:
yeah, agree. This is the legacy code, need discuss more.